lakefile.lean

```diff
@@ -80,6 +80,13 @@ lean_lib docs where
 ## Executables provided by Mathlib
 -/
 
+/-- `lake exe autolabel 150100` adds a topic label to PR `150100` if there is a unique choice.
```

Comment:
It would be good to mention that `gh` is requried to be installed for this to be work since even people contributing to mathlib might not have it / be aware of it (it's not mentioned in our info for contributors, right?)

Maybe it should also be mentioned in the module doc / docstrings in the lean file as well? (and in an error message if gh is missing?)

Reply:
Doing both sounds good to me. (I haven't checked if `gh` needs to be configured some way, perhaps that can be added as well.)

Reply:
Done.

* I did not add an extra error message if `gh` is not installed, but the user will get something like ``unknown command: `gh`.`` (or whatever `bash` says). And I added the note about requiring `gh` to every docstring the user could possibly look at.
* Installing `gh` is just following the official installation instructions (e.g. https://github.com/cli/cli?tab=readme-ov-file#installation) which are basically installing the os-dependent binaries and running `gh auth login` to authenticate, but no special setup.